srcopsmetrics/cli.py

```diff
@@ -112,6 +112,13 @@ def get_entities_as_list(entities_raw: Optional[str]) -> List[str]:
 @click.option(
     "--metrics", "-m", is_flag=True, required=False, help=f"""Launch Metrics Calculation for specified repository.""",
 )
+@click.option(
+    "--merge-kebechet",
```

Comment:
isn't this subcontext of `thoth`? it shoould be separate, so when you use the cli and use `--thoth` you should have two options `--collect-kebchet` and `--merge-kebechet`, wdyt?

Reply:
That does make sense. However, from the Single Responsibility Principle point of view, the `--collect-kebchet` feature is already provided as `--create-knowledge --repository <some_repo> -e PullRequest -e Issue`, and then the `--thoth` is used to calculate any statistics for given repository related and valuable to team Thoth.

Therefore it would probably be best to have just `--merge` option that aggregates everything based on what's been provided (so you could combine it like `--merge -e PullRequest`, or `--merge -o thoth-station -e Issue`, and also `--thoth --merge` for merging all of the repos that were processed with `--thoth` option . So does it make sense, WDYT?

Reply:
Perfect, thanks sir!! 💯

Reply:
aight, changed!